src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,666 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses IntAct data and maps the interaction types to BEL."""
+
+from zipfile import ZipFile
+
+import pandas as pd
+import pybel.dsl
+from protmapper.uniprot_client import get_mnemonic
+from pybel import BELGraph
+from tqdm import tqdm
+from typing import Dict, Iterable, List
+
+from bio2bel.utils import ensure_path
+
+#: Relationship types in IntAct that map to BEL relation 'increases'
+INTACT_INCREASES_ACTIONS = {
+    'psi-mi:"MI:1143"(aminoacylation reaction)',
+    'psi-mi:"MI:0214"(myristoylation reaction)',
+    'proline isomerization reaction',
+    'psi-mi:"MI:0211"(lipid addition)',
+    'isomerization reaction',
```

Comment:
does this one not have a psi-mi code? same for proline isomerization reaction. if not, put a comment next to it saying so

Reply:
I'm in the middle of fixing this right now ;)